docs/builddocs.rst

```diff
@@ -41,10 +41,10 @@ First, install a TeX distribution such as `MacTeX <http://www.tug.org/mactex/>`_
 .. code-block:: bash
 
     # Install deps
-    pip install -U "sphinx==3.1.2" "sphinx-rtd-theme==0.5.0" "nbsphinx==0.7.1" "Pillow==7.2.0"
+    pip install -U sphinx==3.5.4 sphinx-rtd-theme==0.5.1 nbsphinx==0.8.3 matplotlib==3.3.3
 
-    # Patched m2r: https://github.com/sphinx-doc/sphinx/issues/7420
-    pip install -U -q git+https://github.com/intel-isl/m2r@dev#egg=m2r
+    # Patched m2r: https://github.com/sphinx-doc/sphinx/issues/7420#issuecomment-657160798
```

Comment:
Remove comment

Reply:
Perhaps we shall keep this comment since `m2r2` is an unofficial fork or `m2r`.